server/jetty/src/main/java/io/deephaven/server/jetty/JettyBackedGrpcServer.java

```diff
@@ -96,6 +103,57 @@ public JettyBackedGrpcServer(
         // Add an extra filter to redirect from / to /ide/
         context.addFilter(HomeFilter.class, "/", EnumSet.noneOf(DispatcherType.class));
 
+        // If requested, permit CORS requests
+        FilterHolder holder = new FilterHolder(CrossOriginFilter.class);
```

Comment:
Are there considerations where one might _not_ want CORS support? If so, we could consider adding an optional configuration piece in `io.deephaven.server.jetty.JettyConfig`.

Reply:
Fair, though I can't think of any (we don't use browser cookies for anything here, so a malicious website using a user's browser to make calls on their behalf wouldnt really do anything).

My personal vote is that this land as is, then dig in more with parameterizing all of this for the march release (which DnD is almost inevitably going to move to for this release anyway...).

Reply:
LGTM